nemo_text_processing/text_normalization/taggers/measure.py

```diff
@@ -101,6 +102,25 @@ def __init__(self, cardinal: GraphFst, decimal: GraphFst, deterministic: bool =
             + pynutil.insert(" } ")
             + unit_singular
         )
-        final_graph = subgraph_decimal | subgraph_cardinal
+
+        subgraph_cardinal_dash = (
+            pynutil.insert("cardinal { integer: \"")
+            + cardinal.single_digits_graph
```

Comment:
subgraph_decimal_dash will allow for the test case of "7.2-millimeter," and I believe another test case like "7-millimeter" would work here with cardinal.single_digits_graph, but I don't think the following work after this MR:
"70-millimeter bullet"
"12-inch guns"
"21-knot cruising speed"
"550-pound bomb"

I may have missed something when I applied these manually to my local version, but I believe those will all still fail (though decimal versions of them would work eg "70.1-millimeter bullet").

Reply:
```suggestion
            + cardinal.graph
```